packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -53,11 +54,15 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
       associatedEventId: eventId,
     };
 
-    onFormClose();
-    this.setState({ isVisible: false });
-
-    captureFeedback(userFeedback);
-    Alert.alert(text.successMessageText);
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    checkInternetConnection(() => {
+      onFormClose();
+      this.setState({ isVisible: false });
+      captureFeedback(userFeedback);
+      Alert.alert(text.successMessageText);
+    }, () => {
+      Alert.alert(text.errorTitle, text.networkError);
```

Comment:
i: We should use logger to also log the feedback was capture/failed.

Reply:
Good idea 👍 
Added with fe7b14cd6901f944891a8e191f6709a2813d3e7a